pkg/apis/apps/v1alpha1/types.go

```diff
@@ -47,6 +47,7 @@ type ApplicationSpec struct {
 	SyncPolicies []*ApplicationSyncPolicy `json:"syncPolicies"`
 	// Destination defines the destination clusters where the artifacts will be synced.
 	// It can be overriden by the syncPolicies' destination.
+	// And if both the current field and syncPolicies' destination are empty, the application will be deployed directly in the current cluster.
```

Comment:
To be more accurate, current cluster should be the the cluster where kurator resides